exe/appmap-agent-config

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'optparse'
+require 'appmap'
+require 'appmap/command/agent_setup/config'
+
+@options = { config_file: AppMap::DEFAULT_CONFIG_FILE_PATH }
+
+OptionParser.new do |parser|
+  parser.banner = 'Usage: appmap-agent-init [options]'
+
+  description = "AppMap configuration file path (default: #{AppMap::DEFAULT_CONFIG_FILE_PATH})"
```

Comment:
Probably want to update this to indicate that the file will be created by this command, that it won't be overwritten though. Maybe a `-f, --force` option to replace it? That would enable a developer to use it to "reset" the configuration.

Reply:
Good idea. I've added the `--force`/`-f` option to overwrite any existing config.